examples/requestor_proxy/requestor_proxy.py

```diff
@@ -0,0 +1,70 @@
+from quart import Quart
+
+from ya_httpx_client.serializable_request import Request, Response
+from ya_httpx_client.session import Session
+
+#   This is the only "host" we create here, so the name could be exactly anything
+PROVIDER_URL = 'http://provider_http_server'
+
+#   Image hash and the `startup` function define the HTTP server that will be running on the provider,
+#   and thus the final behaviour of our requestor proxy.
+#   Here we use the same image/startup as in the other example (calculator), but any server that would work
+#   in a Dockerfile should be fine.
+IMAGE_HASH = '1f43e06ecc4ef40084efcf57131aa6056c57b5732bef2bcb6a8cdad2'
+def startup(ctx, listen_on):
+    ctx.run("/usr/local/bin/gunicorn", "--chdir", "/golem/run", "-b", listen_on, "calculator_server:app", "--daemon")
+
+#   Number of providers that will be processing requests. NOTE: providers have no shared state, so
+#   numbers >1 make sense only for stateless servers.
+INIT_CLUSTER_SIZE = 1
+
+#   Golem configuration - this will be passed directly to the Golem object
+#   https://handbook.golem.network/yapapi/api-reference#golem-objects
+EXECUTOR_CFG = {'budget': 10, 'subnet_tag': 'devnet-beta.2'}
+
+
+async def init_session():
+    '''
+    Create a Session object.
+    NOTE: it is important to initialize the session in the quart-managed async context,
+          because we need everything to run in the same loop
+    '''
+    session = Session(EXECUTOR_CFG)
+    session.startup(
+        url=PROVIDER_URL,
+        image_hash=IMAGE_HASH,
+        init_cluster_size=INIT_CLUSTER_SIZE,
+    )(startup)
```

Comment:
This looks awkward to me. I understand `startup()` was meant as a decorator. Maybe we need another, similar function that takes the startup function as parameter?